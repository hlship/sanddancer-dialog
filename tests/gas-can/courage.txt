(setup lighting)
(grab #courage)
teleport control
x sheet
search sheet
x piles
look under piles
smell
(grab #scent)
l
smell
search piles
search piles

