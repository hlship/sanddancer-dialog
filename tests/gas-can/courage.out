

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (setup lighting)
Query succeeded: (setup lighting)
> (grab #courage)
Query succeeded: (grab #courage)
> teleport control

Control Center
Some kind of electrical control stuff must have happened here once, but it's
totally trashed now. Like, piles of trash, broken equipment, and junk are
everywhere. But hey, on one pile there's a somewhat serviceable can opener.

You catch a faint whiff of gasoline coming from somewhere.

Back down is the only place to go.

> x sheet
It seems to be harmless.

> search sheet
You root around the trash hopelessly, but there's a lot of it. You don't turn up
anything useful.

> x piles
Lots of construction junk; conduits, broken equipment, metal sheeting, and so
forth in an unruly pile.

> look under piles
You root around the trash hopelessly, but there's a lot of it. You don't turn up
anything useful.

> smell
Something smells very faintly of gasoline.

> (grab #scent)
Query succeeded: (grab #scent)
> l
Control Center
Some kind of electrical control stuff must have happened here once, but it's
totally trashed now. Like, piles of trash, broken equipment, and junk are
everywhere. But hey, on one pile there's a somewhat serviceable can opener.

You can clearly smell gasoline coming from under that sheet of corrugated metal.

Back down is the only place to go.

> smell
Something smells strongly of gasoline.

> search piles
Your sense of smell leads you straight to a sheet of corrugated metal. You
strain to shift it just barely enough to reach inside and pull out a small
plastic gas can. The contents of the can slosh around... you guess there's
enough gas there to get you home, if you can patch up the truck.

Something about the gas can makes you want to brood about road trips through the
desert.

> search piles
You root around the trash hopelessly, but there's a lot of it. You don't turn up
anything else.

> 
